released/packages/coq-sail/coq-sail.0.16/opam

```diff
@@ -0,0 +1,52 @@
+opam-version: "2.0"
+name: "coq-sail"
+version: "0.16"
+maintainer: "Sail Devs <cl-sail-dev@lists.cam.ac.uk>"
+authors: [
+  "Alasdair Armstrong"
+  "Thomas Bauereiss"
+  "Brian Campbell"
+  "Shaked Flur"
+  "Jonathan French"
+  "Kathy Gray"
+  "Robert Norton"
+  "Christopher Pulte"
+  "Peter Sewell"
+  "Mark Wassell"
+]
+homepage: "http://www.cl.cam.ac.uk/~pes20/sail/"
+bug-reports: "https://github.com/rems-project/coq-sail/issues"
+doc: "https://github.com/rems-project/sail/blob/sail2/README.md"
+license: "BSD-3-clause"
+dev-repo: "git+https://github.com/rems-project/coq-sail.git"
+build: [make "-C" "src" "HAVE_OPAM_BBV=yes"]
+install: [make "-C" "src" "install"]
+depends: [
+  "coq" {>= "8.9.0" & <= "8.17.1"}
+  "coq-bbv" {>= "1.1" & <= "1.3"}
+  # The sail package isn't strictly required, but this is the easiest way for
+  # models' opam files to ensure that the version matches
+  "sail" {= version}
+]
```

Comment:
It's not very CI friendly to depend on a package that's not even required for this package. How about the following instead that we use for Ott?
```suggestion
]
conflicts: [
  "sail" { != version }
]
```